tests/unit/date/helper.js

```diff
@@ -0,0 +1,30 @@
+define( [
+	"jquery",
+	"globalize",
+	"globalize/date",
+	"lib/helper"
```

Comment:
It shouldn't work.
 - $ = "jquery"
 - Globalize = "globalize"
 - helper = "globalize/date"
 - undefined = "lib/helper"